tidy3d/components/scene.py

```diff
@@ -107,8 +110,19 @@ class Scene(Tidy3dBaseModel):
         (),
         title="Structures",
         description="Tuple of structures present in scene. "
-        "Note: Structures defined later in this list override the "
-        "simulation material properties in regions of spatial overlap.",
+        "Note: In regions of spatial overlapping between structures, "
+        "material properties are dictated by structure of higher priority "
+        "defined by ``overlapping_priority``.",
+    )
+
+    overlapping_priority: OverlappingPriority = pd.Field(
+        "latter",
+        title="Resolution Strategy In Structure Overlapping Region",
+        description="In regions of spatial overlapping between structures, "
+        "if ``overlapping_priority=latter``, structures defined later in the list haver higher "
+        "priority. If ``overlapping_priority=metal``, metallic structures of higher conductivity "
```

Comment:
By `metallic` we explicitly mean PEC or SIBC but not e.g. usual dispersive metals?

Reply:
we don't have any special subpixel methods applied to usual dispersive metals. So we only mean "good conductor" by "metallic" here.

Reply:
Now it's renamed to conductor